gridtools_lib/pylib/make_topog/Topog.py

```diff
@@ -0,0 +1,76 @@
+from typing import List
```

Comment:
could we have this file live in gridtools/make_topog?  I'm thinking we should keep gridtools_lib/pylib to just ctypes wrappers to C